docs/sources/next/javascript-api/k6-html/element/element-selection.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Element.selection()'
+excerpt: 'Retrieve the Selection matching this element.'
```

Comment:
```suggestion
description: 'Retrieve the Selection matching this element.'
```

Reply:
Sorry for the late reply @heitortsergent :bow: 

This is what I just copied from a different page around this docs.

Is this the new name - and we should rename them all, or is it just an alias you prefer? Or something else?

Reply:
@mstoykov no worries! We should rename them all, it's something I missed during the migration. 😞 

If you can please just change it in this PR, I can open a separate PR to change it in all other files.